components/syncer/kcp-unstable/appstudio-internal-syncer.yaml

```diff
@@ -0,0 +1,221 @@
+  name: kcp-syncer-appstudio-internal-jbrk5z54
+subjects:
+- kind: ServiceAccount
+  name: kcp-syncer-appstudio-internal-jbrk5z54
+  namespace: kcp-syncer-kcp-unstable
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kcp-dns-appstudio-internal-jbrk5z54
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kcp-dns-appstudio-internal-jbrk5z54
+subjects:
+  - kind: ServiceAccount
+    name: kcp-dns-appstudio-internal-jbrk5z54
+    namespace: kcp-syncer-kcp-unstable
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kcp-syncer-appstudio-internal-jbrk5z54
+  namespace: kcp-syncer-kcp-unstable
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: kcp-syncer-appstudio-internal-jbrk5z54
+  template:
+    metadata:
+      labels:
+        app: kcp-syncer-appstudio-internal-jbrk5z54
+    spec:
+      containers:
+      - name: kcp-syncer
+        command:
+        - /ko-app/syncer
+        args:
+        - --from-kubeconfig=/kcp/kubeconfig
+        - --sync-target-name=appstudio-internal
+        - --sync-target-uid=215a89f0-828f-4703-8992-e34769def7d3
+        - --from-cluster=root:redhat-appstudio-internal-compute
+        - --api-import-poll-interval=1m0s
+        - --resources=configmaps
+        - --resources=deployments.apps
+        - --resources=routes.route.openshift.io
+        - --resources=secrets
+        - --resources=services
+        - --resources=statefulsets.apps
+        - --qps=20
+        - --burst=30
+        - --dns=kcp-dns-appstudio-internal-jbrk5z54.kcp-syncer-kcp-unstable.svc.cluster.local
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: ghcr.io/kcp-dev/kcp/syncer:main
+        imagePullPolicy: Always
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - name: kcp-config
+          mountPath: /kcp/
+          readOnly: true
+      serviceAccountName: kcp-syncer-appstudio-internal-jbrk5z54
+      volumes:
+        - name: kcp-config
+          secret:
+            secretName: kcp-syncer-appstudio-internal-jbrk5z54
+            optional: false
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kcp-dns-appstudio-internal-jbrk5z54
+  namespace: kcp-syncer-kcp-unstable
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: kcp-dns-appstudio-internal-jbrk5z54
+  template:
+    metadata:
+      labels:
+        app: kcp-dns-appstudio-internal-jbrk5z54
+    spec:
+      containers:
+      - name: kcp-dns
+        command:
+        - /ko-app/syncer
+        args:
+        - dns
+        - start
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: ghcr.io/kcp-dev/kcp/syncer:main
+        imagePullPolicy: IfNotPresent
```

Comment:
should be Always or we should bind to specific tag.

Reply:
good catch - forgot about the second deployment 3c341b2